perception/navigator_vision/nodes/object_classifier.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import txros
+from txros import util
+from twisted.internet import reactor, defer
+from object_classification import LidarToImage, HOGDescriptor, SVMClassifier
+
+
+class ObjectClassifier(object):
+
+    def __init__(self, nh, descriptor, classifier):
+        self.nh = nh
+        self.descriptor = descriptor
+        # load up the trained svm via pickle
+        self.classifier = classifier
+
+    @util.cancellableInlineCallbacks
+    def init_(self):
+        self.lidar_to_image = yield LidarToImage(self.nh, self.image_cb).init_()
+        defer.returnValue(self)
+
+
+    def image_cb(self, imgs):
+        for img in imgs:
+            objid = img["id"]
+            img = img["img"]
+            desc = self.descriptor.get_descriptor(img)
+            classification = self.classify(desc)
+            print "id", objid, classification
```

Comment:
yes, will be implemented. You can put it in the issue that I made for it for now.